src/studio/src/designer/frontend/dashboard/features/standaloneDataModelling/DataModelling.test.tsx

```diff
@@ -6,6 +6,21 @@ import {LoadingState} from 'app-shared/features/dataModelling/sagas/metadata';
 import {renderWithProviders} from "test/testUtils";
 import {setupStore} from "app/store";
 
+// workaround for https://jestjs.io/docs/26.x/manual-mocks#mocking-methods-which-are-not-implemented-in-jsdom
```

Comment:
I see this is added to several tests, but I would suggest moving this to the `setupTests` file instead. Then it will be enabled everywhere, and theres only 1 place to remove it from when/if JSDOM adds support for this (or when/if we switch to something else which has support, like [Happy-DOM](https://github.com/capricorn86/happy-dom))

Reply:
Mm good point!